src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const people = document.querySelector('ul');
+const peopleList = [...document.querySelectorAll('li')];
+
+function toNumber(string) {
+  return +string.slice(1).replace(',', '');
+};
+
+function sortlist(list) {
+  list.sort((a, b) => toNumber(b.dataset.salary) - toNumber(a.dataset.salary));
+
+  people.append(...list);
```

Comment:
Do not change the data that comes to the function

```suggestion
   const sortedList = [...list].sort((a, b) => (
       toNumber(b.dataset.salary) - toNumber(a.dataset.salary)
     );

  people.append(...sortedList);
```